tox.ini

```diff
@@ -126,13 +126,14 @@ deps =
     profile: py-cpuinfo==8.0.0
     # TODO: remove py dependency once https://github.com/ionelmc/pytest-benchmark/pull/227 is released
     profile: py
-    py{35,36,37,38,39,310}-profile: pytest-asyncio
+    py{35,36,37,38,39,310,311}-profile: pytest-asyncio
     py{27,35,36,37,38,39}-profile: uwsgi; sys_platform != 'win32'
-    py{27,35,36,37,38,39,310}-profile: gunicorn; sys_platform != 'win32'
-    py{27,35,36,37,38,39,310}-profile-gevent: gunicorn[gevent]; sys_platform != 'win32'
+    py{27,35,36,37,38,39,310,311}-profile: gunicorn; sys_platform != 'win32'
+    py{27,35,36,37,38,39,310,311}-profile-gevent: gunicorn[gevent]; sys_platform != 'win32'
     py{27,35,36,37,38,39}-profile-minreqs: protobuf==3.0.0
-    py{310}-profile-minreqs: protobuf==3.8.0
+    py3{10,11}-profile-minreqs: protobuf==3.8.0
     profile-minreqs: tenacity==5.0.1
```

Comment:
Can't find the option to suggest change 😢 

Can we specify `py{27,35,36,37,38,39,310}-profile-minreqs: tenacity==5.0.1` here so that this doesn't interfere with the double requirement of `tenacity==7.0.0` for python 3.11?

Reply:
Done!